research/package.json

```diff
@@ -3,6 +3,7 @@
   "description": "Open UI documentation.",
   "version": "1.0.0",
   "author": "Open UI",
+  "type": "module",
```

Comment:
Why did you add "type: module" and prettier config for `mjs` files? mjs shouldn't exist if scripts are modules by default.

Reply:
That one was actually nasty. Adding `hastscript` broke build for me. I have NodeJS 19, so that's probably because of that. Other option which was available to me was rename `astro.config.js` to `astro.config.mjs`, but that trigger red squiggles in VS Code which like to apple different code style. I do not know how to disable that and decide that this was least invasive path. I may be wrong here.

Reply:
I worry about others having similar issues when cloning the repo and running it locally. As long as it doesn't worsen the DX, it shouldn't be a problem.